Terminal/user.css

```
@@ -0,0 +1,136 @@
+}
+
+.player-controls__buttons {
+	transform: translateY(6px);
+}
+
+.main-playPauseButton-button {
+	background-color: var(--spice-main);
+	box-shadow: var(--spice-shadow) 0 5px 9px 0px;
+	--button-size: 50px !important;
+	color: var(--spice-text);
+	background-color: var(--spice-player) !important;
```

Biome (2.4.7)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

(lint/suspicious/noDuplicateProperties)